flumine/backtest/utils.py

```diff
@@ -42,3 +42,35 @@ def __init__(
         self.size_cancelled = size_cancelled
         self.cancelled_date = cancelled_date
         self.error_code = error_code
+
+
+class SimulatedUpdateResponse:
+    def __init__(
+        self,
+        status: str = None,
+        bet_id: str = None,
+        new_persistence_type: str = None,
+        updated_date: datetime = None,
+        error_code: str = None,
+    ):
+        self.status = status
+        self.bet_id = bet_id
+        self.new_persistence_type = new_persistence_type
+        self.updated_date = updated_date
+        self.error_code = error_code
+
+
+class SimulatedReplaceResponse:
```

Comment:
This is a bit different as it contains a place response and cancel response 

https://github.com/liampauling/betfair/blob/master/betfairlightweight/resources/bettingresources.py#L1067